rules-reviewed/eap7/weblogic/weblogic-xml-descriptors.windup.xml

```diff
@@ -88,7 +88,9 @@
                 </or>
             </when>
             <perform>
+                <!--
                 <classification title="WebLogic EJB XML" category-id="mandatory" effort="0" />
+                -->
```

Comment:
@OndraZizka  but you are effectively adding that classification or I don't get the comment and PR title ;)